ksql-common/src/test/java/io/confluent/ksql/util/SchemaUtilTest.java

```diff
@@ -376,6 +377,20 @@ public void shouldGetCorrectSqlType() {
     assertThat("Invalid SQL type.", sqlType7, equalTo("MAP<VARCHAR,DOUBLE>"));
   }
 
+  @Test
+  public void shouldGetCorrectSqlTypeFromSchemaType() {
+    assertThat(SchemaUtil.getSchemaTypeAsSqlType(Schema.Type.STRING), is("VARCHAR(STRING)"));
+    assertThat(SchemaUtil.getSchemaTypeAsSqlType(Schema.Type.INT64), is("BIGINT"));
+    assertThat(SchemaUtil.getSchemaTypeAsSqlType(Schema.Type.INT32), is("INTEGER"));
+    assertThat(SchemaUtil.getSchemaTypeAsSqlType(Schema.Type.FLOAT64), is("DOUBLE"));
+    assertThat(SchemaUtil.getSchemaTypeAsSqlType(Schema.Type.BOOLEAN), is("BOOLEAN"));
+  }
+
+  @Test(expected = IllegalArgumentException.class)
```

Comment:
This won't be needed after merging https://github.com/confluentinc/ksql/pull/1401.